syntax="proto3";
// 指定模板类的包路径
option java_package = "com.szlongzy.common.protobuf.dto";
// 指定模板类的名称，名称必须是有实际业务意义的
option java_outer_classname = "RebootVo";


/* 重启请求*/
message rebootRequest{

  // 实际目标设备
  string  objDevId = 1;
  //备用
  string authKey =2;
  string uid = 3;
}

/* 重启设备回复*/
message rebootResponse{
  // 回复
  bool  result = 1;
  // 实际目标设备
  string  objDevId = 2;
  // 错误信息
  string  msg = 3;
}